WeakAuras/BuffTrigger2.lua

```diff
@@ -837,26 +837,55 @@ local function RemoveState(triggerStates, cloneId)
   end
 end
 
-local function GetAllUnits(unit, allUnits)
+local function GetAllUnits(unit, allUnits, includePets)
```

Comment:
true, false, nil are 3 values, but we should avoid using it as a tristate. Even though we do use it in some places, those usages lead to rather poor code.

Reply:
Actually this is due to the options code. Argh! Then it's probably okayish.